### DISCUSSION TITLE
Closing a PR opened manually via dependency dashboard

### DISCUSSION THREAD

**Discussion by benasher44** [renovatebot/renovate#13565] [Request Help] [answered]
Hello! What is the proper way to close a PR opened manually by ticking a box in the dependency dashboard? I tried just closing it, and renovate opened a new one. After that, I closed it and then quickly followed up by removing that line from the dependency dashboard, which seemed to work. Is there a more straightforward way that I should do next time? Thanks!

**Comment by rarkins** [comment]
This shouldn't normally happen because Renovate unchecks the checkbox at the end of the run, and no special handling is required. Perhaps somehow the checkbox remained checked or got rechecked so that the second time Renovate ran it still saw the dashboard checked.

You're welcome to look into the debug logs for each run if you want to treat it as a possible bug, but otherwise you shouldn't worry about it.

**Reply by benasher44** [reply]
At the time, the dependency had been moved to the "open" section of the dependency dashboard, and it was unchecked. I'll enable debug logs next time in case it happens again, and the I'll file a bug.

**Reply by benasher44** [reply]
Thanks for following up!

**Comment by benasher44** [comment, marked as answer]
Turns out the PR was "immortal" because it came from a pre-defined grouping. After some more reading on the subject, I understand what's going on now. Just wanted to add this follow up. Thanks again!
